src/app/components/libraries/dialogs/add-slr-dialog/add-slr-dialog.component.ts

```diff
@@ -0,0 +1,199 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import {
+  MAT_DIALOG_DATA,
+  MatDialog,
+  MatDialogRef,
+} from '@angular/material/dialog';
+import {
+  AbstractControl,
+  FormArray,
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  Validators,
+} from '@angular/forms';
+import { StudyDto } from 'api-library/models/study-dto';
+import { Study } from 'api-library/models/study';
+
+@Component({
+  selector: 'app-add-slr-dialog',
+  templateUrl: './add-slr-dialog.component.html',
+  styleUrls: ['./add-slr-dialog.component.scss'],
```

Comment:
If you don't need stlye files, delete them.

Reply:
deleted